src/core/class/GroupViewTools.h

```diff
@@ -7,47 +7,78 @@
 namespace SEMBA {
 namespace Class {
 namespace Group {
-namespace View {
-
-    std::vector<const Geometry::ElemR*> filterView(
-        const std::vector<const Geometry::ElemR*>& view,
-        const std::function<bool(const Geometry::ElemR*)>& map
-    ) {
-        std::vector<const Geometry::ElemR*> res;
-
-        std::copy_if(
-            view.begin(),
-            view.end(),
-            std::back_inserter(res),
-            map
-        );
-
-        return res;
-    }
-
-    // TODO: To update Structured mesh and consider using Integer inside and communicate with Real outside
-    std::vector<const Geometry::ElemR*> castToReal(
-        const std::vector<const Geometry::Elem*>& view
-    ) {
-
-        for (const auto& elem : view) {
-            if (!elem->is<Geometry::ElemR>()) {
-                throw std::logic_error("View contains elements of type different than Geometry::ElemR");
+    class View {

```

Comment:
I don't see the point of changing the namespace to a class if all methods are going to be static. This would be interesting if we want to treat the views as an object with some special capabilities.
Right now we are basically losing two things: 
1.  We have to explicitly say the class the static method belongs to. Classes can not be used as namespaces to give context.
2. The way is currently implemented, all the methods are included in the header. This can create an overhead on the compilation times as this class is used very often.

 I think I wouldn't do this at this point, but we could discuss it. There might be some benefits.

Reply:
Agreed. The change is motivated to get SEMBA to compile as fast as possible. There were errors due to duplicate definitions of the namespace methods when compiling. 

I will mark this as a temporary work around. As you suggest, probably better to follow the `.h` and `.hpp` schema as a mid solution. After that, we can think about giving `template` to this piece of code.